boards.txt

```diff
@@ -228,7 +228,7 @@ Generic_nRF51822.build.board=GENERIC
 Generic_nRF51822.build.core=nRF5
 Generic_nRF51822.build.variant=Generic
 Generic_nRF51822.build.variant_system_lib=
-Generic_nRF51822.build.extra_flags=-DNRF51
+Generic_nRF51822.build.extra_flags=-DNRF51 -DBOARD_PCA20006 -DNRF51822
```

Comment:
Ohh yes, that was there just for Debug while I built the other Board! thanks for the advice!